packages/website/app/components/articleBox/articleBox.tsx

```diff
@@ -0,0 +1,99 @@
+import { Link } from "@remix-run/react";
+import { LinksFunction } from "@remix-run/server-runtime";
+import { AiFillCalendar, AiFillDatabase, AiFillHeart } from "react-icons/ai";
+
+import AvatarUrl from "../../temporary/avatar.jpg";
+import  '~/styles/articleBox.css'
+// const Container = styled.div`
+//   width: 100%;
+//   height: 30vh;
+//   display: grid;
+//   grid-template-rows: 20% 80%;
+//   .top {
+//     h1 {
+//     display: inline-block;
+//       font-size: 50px;
+//       a{
+//         font-size: 50px;
+//         &:hover{
+//             background-color: transparent;
+//         }
+//       }
+//       &::after{
+//         display: block;
+//         content: "";
+//         width: 0%;
+//         height: 3px;
+//         background-color: var(--gray);
+//         transition: all 0.3s;
+//       }
+//       &:hover::after{
+//         width: 100%;
+//       }
+//     }
+//   }
+//   .bottom {
+//     display: grid;
+//     grid-template-columns: 30% 70%;
+//     .left {
```

Comment:
Nested css can be implemented by variants, FYI, https://v2.tailwindcss.com/docs/configuring-variants.

Another powerful feature is customization. The ability to extend existed utilities and use JS generate css utilities:

https://github.com/io-club/website/blob/5803bb57a6d291669f52f2323b9643eec2ff7386/packages/website/windi.config.js#L91-L93

https://github.com/io-club/website/blob/5803bb57a6d291669f52f2323b9643eec2ff7386/packages/website/windi.config.js#L128-L168